### ISSUE TITLE
Feature request: int(x), frac(x), floor(x) functions? 

### ISSUE THREAD

**Issue by ifmihai** [tranleduy2000/ncalc#32]
I tried to create a new function 
Modc(x) = mod(x, 360)

But doesn't work. 
Either i did a mistake, 
Or only basic functions can be used when defining a new function. 

If only basic functions are permitted, 
Then these 3 functions should be added. 

Also an if() function should be useful, 
But maybe there's one, and i missed it.

**Comment by axkr** [comment]
In Version 3.4.1 in IDE mode write `modc(x_):=mod(x,360)` and evaluate the expression.
After that you can evaluate `modc(a)` to get the result `mod(a,360)`

**Comment by axkr** [comment]
In Version 3.4.1 in IDE mode try 
```
    IntegerPart(42.5) -> 42
    FractionalPart(42.5) -> 0.5
    Floor(-29/13) -> -3

    if(1<2, a, b) -> a
```

**Comment by axkr** [comment]
Fixed with 3.4.3-beta
